tasks/zadanie_12.php

```diff
@@ -13,5 +13,12 @@
 
 $number = $params[0]; // tej linijki nie ruszamy :)
 
-echo "5 4 3 2 1";
-
+for ($i = 1; $i <= 5; $i++) {
```

Comment:
spróbuj tutaj w drugą stronę ;)
```suggestion
for ($i = $number; $i >= 1; $i--) {
```